lib/pf/services/manager/dhcpd.pm

```diff
@@ -86,6 +87,31 @@ EOT
     return 1;
 }
 
+
+=head2 omapi_section
+
+Generate the omapi section if it is defined
+
+=cut
+
+sub omapi_section {
+    return '' if isdisabled($Config{advanced}{use_omapi_to_lookup_mac});
```

Comment:
How would that be useful?
It just add a comment to a generated config file.
We already know it is not enabled by it being disabled in pf.conf